docs/getting_started/tutorial06.rst

```diff
@@ -0,0 +1,99 @@
+Your First CodeRed Website: Part 6 -- Classifiers in CMS
+========================================================
+
+Now we actually want to go back to our blog posts and organize them. We can do 
+that by adding **Classifiers** to our pages so that we can access them via their
+category. 
+
+What are Classifiers?
+---------------------
+
+Classifiers are essentially categories that you give to an item. For our sweets shop,
+classifiers that we might use could be something like:
+
+* **Flavors:** Strawberry, Vanilla, Chocolate, etc.
+
+* **Types of baked goods:** Cupcakes, Cookies, Muffins, Cinnamon Rolls, etc.
+
+* **Types of content:** News, Press, Interviews, Recipes, etc.
+
+Adding Classifiers in the CMS
+-----------------------------
+
+There are two ways to add Classifiers. The first way can be found by going to **Snippets > Classifiers**. 
+Select that you want to add a new Classifier, and you would do this for each Classifier that you want to add.
+For the purposes of this tutorial, we are going to add a Content Types classifier and a Baked Goods classifier. 
+
+.. figure:: img/tutorial_new_classifier.png
+    :alt: Screen for adding a new Classifier.
+
+    The admin screen for adding a new Classifier.
+
+The first Classifier we are adding is the Baked Good ones. For Name, we would put Baked Goods because this is the
+generalized heading for the category. Then we would click the **Add Classifier Terms** button to begin adding the
+sub-categories below that main term. 
+
+.. figure:: img/tutorial_edit_classifiers.png
+    :alt: The Classifiers that we added.
+
+    The edit screen for the Classifiers that we added.
+
+.. note::
+    Once you've added Classifiers in Snippets, there's no need to add Classifiers the other way. In fact,
+    the option to add them the other way is no longer available. 
```

Comment:
I'm not following this note... I thought there was only one way?

Reply:
Technically there is only one way but if you were in a page and thought, "I need a classifier," you can click to add one in the Classifier tab vs leaving the page to go to Snippets > Classifiers. But I think it just takes you to the Snippets area to add a Classifier. It's more of a shortcut than a different way of doing it. So I'll have to adjust that.

Reply:
Removed the note and cleared it up a bit.